src/Vegetation/canopy_parameterizations.jl

```diff
@@ -445,3 +450,86 @@ function medlyn_conductance(
     gs = g0 + Drel * medlyn_term * (An / ca)
     return gs
 end
+
+"""
+    function bulk_SW_albedo(
+                            ρ_leaf_sw::FT,
+                            α_soil::FT,
+                            K::FT,
+                            LAI::FT,
+                            Ω::FT,
+                            ) where {FT}
+
+Computes albedo                            
+"""
+function bulk_SW_albedo(
+    ρ_leaf_sw::FT,
+    α_soil::FT,
+    K::FT,
+    LAI::FT,
+    Ω::FT,
+) where {FT}
+    α_SW = α_soil * exp(-K * LAI * Ω) + ρ_leaf_sw * (1 - exp(-K * LAI * Ω))
+    return α_SW 
+end
+
+"""
+    function bulk_LW_emissivity(
+                            ρ_leaf_sw::FT,
+                            α_soil::FT,
+                            K::FT,
+                            LAI::FT,
+                            Ω::FT,
+                            ) where {FT}
+
+Computes emissivity                            
+"""
+function bulk_LW_emissivity(
+    LAI::FT,
+    LAI_max::FT,
+    k_ϵ::FT,
+    fsnow::FT,
+    fground::FT,
+    ϵ_soil::FT,
+    ϵ_veg::FT,
+    ϵ_snow::FT,
+) where {FT}
+    ε_lai = (LAI/LAI_max)^(1/k_ϵ) * ε_veg             # Emissivity of vegetation as a function of LAI
+    ε_p = fsnow * ε_snow + (1.0 - fsnow) * ε_lai      # Emissivity of the snow-covered or vegetated surface
+    ε_LW = (1.0 - fveg) * ε_soil + fveg * ε_p         # Land surface longwave emissivity
+    return ϵ_LW  
+end
+ 
+"""
+    penman_monteith(
```

Comment:
possibly: change formula to match Bonan, and then cite his book? it should give the same answer